media/js/fonts.js

```diff
@@ -111,12 +111,12 @@
             var fontAttribute = index;
             var fontLoaded = true;
             $.each(this, function() {
-                if(this == false) {
+                if(!this) {
```

Comment:
Right now it loops through each font and checks if it is loaded, changing `fontLoaded` to false if it hasn't. If we just do `fontLoaded = this` then `fontLoaded` will have the value of the last font in the loop, won't it? Or am I misunderstanding where we'd insert it?